startup/BMM/workspace.py

```diff
@@ -59,8 +60,7 @@ def initialize_workspace():
 
 def check_profile_branch():
     here = os.getcwd()
-    STARTUP = os.path.dirname(BMM.functions.__file__)
-    os.chdir(get_ipython().profile_dir.location)
+    os.chdir(os.path.split(startup_dir)[0])
```

Comment:
I suggest using `os.path.dirname` here too:
```suggestion
    os.chdir(os.path.dirname(startup_dir))
```

Reply:
OK